.gitlab/ci/.gitlab-ci.on-push.yml

```
@@ -787,11 +799,39 @@ test-upload-flow:
     reports:
       junit:
         - "${ARTIFACTS_FOLDER}/test_playbooks_report.xml"
+        - "${ARTIFACTS_FOLDER}/e2e_tests_result.xml"
     paths:
       - "${ARTIFACTS_FOLDER}/test_playbooks_report.xml"
+      - "${ARTIFACTS_FOLDER}/e2e_tests_result.xml"
       - ${CI_PROJECT_DIR}/artifacts/*  # restoring the default artifacts path from the job default settings
       - ${CI_PROJECT_DIR}/pipeline_jobs_folder/*  # restoring the default artifacts path from the job default settings
 
+
+.e2e_test_results:
+  stage: results
+  rules:
+    - !reference [ .filter-non-nightly-docker-updates-rule, rules ]
+    - if: '$CI_PIPELINE_SOURCE =~ /^(push|contrib)$/'
```

```suggestion
    - if: '$NIGHTLY'
```

change when finishing up testing